ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -124,10 +124,13 @@ internal class PaywallViewModelImpl(
     override fun restorePurchases() {
```

Comment:
It would be nice to have tests for `PaywallViewModel` too (example: https://github.com/RevenueCat/purchases-ios/blob/main/Tests/RevenueCatUITests/Purchasing/PurchaseHandlerTests.swift#L55)

Reply:
Absolutely we should. I'm taking note so I will add them tomorrow.